exercises/practice/high-scores/high_scores.odin

```diff
@@ -0,0 +1,34 @@
+//
+// This is only a SKELETON file for the 'High Scores' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+package high_scores
+
+import "core:slice"
```

Comment:
I wonder if this should be removed so that students can discover it themselves. After all, they'll see examples of it in the tests.